### ISSUE TITLE
sneakers.io expired?

### ISSUE THREAD

**Issue by boone** [jondot/sneakers#451]
Currently sneakers.io leads to a page offering the domain for sale. Can it be reactivated? If not, that domain link should be removed/updated on the Github project About section as well as the gemspec.

Thanks!

**Comment by jondot** [comment]
Yep, unfortunately someone exploited a window of opportunity and snagged it, I've been battling it since. I'm working on replacing the domain.

**Comment by boone** [comment]
Sorry that happened. Thanks for the update!
